rentme/src/components/propertyOwner/PropertyCard/ServiceRequests.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState, useEffect} from "react"
+import ServiceRequest from './ServiceRequest'
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  cardContainer: {
+    display: "flex",
+    width: "75%",
+    justifyContent: "space-around",
+    flexWrap: "wrap"
+  }
+})
+
+
+const ServiceRequests = () => {
+
+  const classes = useStyles()
+
+  return (
+    <div className={classes.cardContainer}>
```

Comment:
Temporarily rendering multiple components for display testing. 
This will be a map of all service requests linked to a property when finished